From a74ace07b68177a5d5fbcecbfea5f43b364ef68d Mon Sep 17 00:00:00 2001
From: Xing Zheng <zhengxing@rock-chips.com>
Date: Wed, 4 Jul 2018 20:38:00 +0800
Subject: [PATCH] ARM: dts: rockchip: enable 'rockchip,no-hp-det' for rk3308
 voice modules

There isn't headphone on all of rk3308 voice modules,
so we don't need to enable headphone detection.

Change-Id: I526dc02dfb3c0b92503c8a7821baa82b87c8c996
Signed-off-by: Xing Zheng <zhengxing@rock-chips.com>
---
 arch/arm/boot/dts/rk3308-voice-module-v10-aarch32.dtsi | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm/boot/dts/rk3308-voice-module-v10-aarch32.dtsi b/arch/arm/boot/dts/rk3308-voice-module-v10-aarch32.dtsi
index 1a9fada7de48..ef8c46537693 100644
--- a/arch/arm/boot/dts/rk3308-voice-module-v10-aarch32.dtsi
+++ b/arch/arm/boot/dts/rk3308-voice-module-v10-aarch32.dtsi
@@ -181,6 +181,7 @@
 &acodec {
 	status = "disabled";
 
+	rockchip,no-hp-det;
 	rockchip,loopback-grp = <1>;
 	spk-ctl-gpios = <&gpio0 RK_PA5 GPIO_ACTIVE_HIGH>;
 };
-- 
2.35.3

